packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -221,6 +226,17 @@ contract SortedOracles is ISortedOracles, ICeloVersionedContract, Ownable, Initi
     return (false, oldest);
   }
 
+  /**
+   * @notice Sets the equivalent token for a token.
+   * @param token The address of the token.
+   * @param equivalentToken The address of the equivalent token.
+   */
+  function setEquivalentToken(address token, address equivalentToken) external onlyOwner {
```

Comment:
it should be possible to remove this, otherwise if a (real) oracle of the mapped token is added, it would not have any effect.

Reply:
It was possible to remove equivalentToken by setting it to address(0), but for better transparency I introduced deleteEquivalentToken function now